landing/src/index.css

```
@@ -0,0 +1,88 @@
+@import 'https://fonts.googleapis.com/css2?family=JetBrains+Mono:wght@400;500;700&display=swap';
+@import "tailwindcss";
+
+@theme {
+  --font-sans: "Neue Haas Grotesk Display Pro", "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Stylelint (17.11.0)

[error] Expected "Helvetica" to be "helvetica" (value-keyword-case)

(value-keyword-case)